### ISSUE TITLE
[canary] NPC "Memorandoll" not showing up

### ISSUE THREAD

**Issue by eyes-and-brain** [topaz-next/topaz#2501]
- [ x] I have searched existing issues (https://github.com/topaz-next/topaz/issues) to see if the issue has already been opened
- [x ] I have checked the commit log to see if the issue has been resolved since my server was last updated
- [ x] This issue occurs on `release` branch (if other branch, please specify)

**_Additional Information_** (Steps to reproduce/Expected behavior) **:** 

Talk to NPC "Isakoth" to start the "Records of Eminence".
During the event, the NPC "Memorandoll" is not showing up.

The log of topaz_game.exe shows the following warning
[Warning] Server needs NPC <17739954>.

**Comment by cocosolos** [comment]
Looks like NPC not captured.
`npc_list.sql`
```
-- NC: INSERT INTO `npc_list` VALUES (17739954,'NOT_CAPTURED','???',0,0.000,0.000,0.000,0,50,50,0,0,0,0,0,0x0000320000000000000000000000000000000000,0,NULL,0);
```
I'm guessing this is the NPC in the other zones
```
INSERT INTO `npc_list` VALUES (17764604,'qm3','???',0,0.000,0.000,0.000,0,40,40,0,0,0,6,27,0x0000CA0300000000000000000000000000000000,32,NULL,1);
INSERT INTO `npc_list` VALUES (17719639,'qm9','???',0,0.000,0.000,0.000,0,50,50,0,0,0,6,27,0x0000CA0300000000000000000000000000000000,32,NULL,0);
```
So could be added without a proper capture. Not sure why the speed and widescan is different with the other 2, I'm guessing one of them is wrong but I don't think it matters for this NPC.

**Comment by zach2good** [comment]
Anyone who wants to capture this, the area is `{ Bastok Markets }`

**Comment by eyes-and-brain** [comment]
Thank you for your comment.
I was able to understand.

**Comment by wrenffxi** [comment]
closed via #2620 merging into release
